### PR TITLE
Update user cases

### PR DESCRIPTION
**Pull request by jhqiu21**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -312,32 +312,196 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case resumes at step 1. 
+
+
+**Use case: UC06 - List Contact**
+
+**MSS**
+
+1. Talent Manager requests to list persons
+2. System shows a list of persons
```
Comment (wrjgold) [inline docs/DeveloperGuide.md:422]: Same for these portions, specify we're listing all

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -312,32 +312,196 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case resumes at step 1.
+
+* 2a. If no events are scheduled.
+
+  * 2a1. System displays a message stating "No events.".
+
+    User case ends.
+
+
+**Use case: UC09 - Delete Events**
```
Comment (wrjgold) [inline docs/DeveloperGuide.md:483]: Should we abstract out like about into delete after list, in case we have similar find event feature in the future?

**Review by wrjgold — COMMENTED** [review]
looks good, some minor suggestions

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -312,32 +312,196 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. Talent Manager [find contacts (UC05)](#)
+2. Talent Manager [delete contact (UC04)](#)
+
+   Use case ends.
+
+**Use case: UC04 - Delete Contact**
+
+**MSS**
+
+1. Talent Manager requests to delete a specific person in the list
```
[the remark this answers is not in the document]
Comment (jhqiu21) [reply docs/DeveloperGuide.md:368]: That is what I am worrying about, I think "by index" may involve some implementation details.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -312,32 +312,196 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case: UC02 - Delete Contact after List**
+
+**MSS**
+
+1. Talent Manager [list contacts (UC06)](#)
+2. Talent Manager [delete contact (UC04)](#)
+
+   Use case ends.
+
+**Use case: UC03 - Delete Contact after List**
```
[the remark this answers is not in the document]
Comment (jhqiu21) [reply docs/DeveloperGuide.md:355]: Yes sure, this is a typo. I will fix it.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -312,32 +312,196 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case resumes at step 1.
+
+* 1b. The keyword is empty.
+
+  * 1b1. System shows an error message.
+
+    Use case resumes at step 1. 
+
+
+**Use case: UC06 - List Contact**
```
[the remark this answers is not in the document]
Comment (jhqiu21) [reply docs/DeveloperGuide.md:417]: Yes, sure. I will add it to specify.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jhqiu21) [reply docs/DeveloperGuide.md:422]: Fixed

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jhqiu21) [reply docs/DeveloperGuide.md:483]: I think that is a good attempt. I will abstract it like `delete contact`.

**Review by fusin02 — APPROVED** [review]

**Review by wrjgold — APPROVED** [review]

**Pull request merged**